### PR TITLE
fix: fields for cloudfront logging

### PR DESCRIPTION
**Pull request by akshaymagapu**
Update fields as per logs for cloudfront

### CODE REVIEW

File: test/audits/sitemap-product-coverage.test.js [hunk 1]
```diff
@@ -848,7 +848,9 @@ describe('Sitemap Product Coverage Audit', () => {
       expect(generateOpportunity).to.be.a('function');
     });
 
-    it('should handle configuration edge cases', async () => {
+    it('should handle configuration edge cases', async function () {
+      // korablov need to look at this test
+      this.timeout(5000);
```
Comment (akshaymagapu) [inline test/audits/sitemap-product-coverage.test.js:853]: 1 failing
  1) Sitemap Product Coverage Audit
       Edge Cases and Error Handling
         should handle configuration edge cases:
     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/runner/work/spacecat-audit-worker/spacecat-audit-worker/test/audits/sitemap-product-coverage.test.js)
      at listOnTimeout (node:internal/timers:588:17)
      at process.processTimers (node:internal/timers:523:7)
